Client/Frontend/ContextualHint/ContextualHintViewController.swift

```diff
@@ -161,7 +161,7 @@ class ContextualHintViewController: UIViewController, OnViewDismissable, Themeab
         hintView.configure(viewModel: viewModel)
         applyTheme()
 
-        if viewProvider.shouldPresentContextualHint() && shouldStartTimer {
```

Comment:
I have a note here:
- It does not make sense to check viewProvider.shouldPresentContextualHint() and shouldStartTimer together in the configure function from ContextualHintViewController. Just checking shouldStartTimer is sufficient because the startTimer method first checks shouldPresentContextualHint() before continuing the function. I hope the message it's understandable. :))